tests/queries/test_q.py

```diff
@@ -3,6 +3,30 @@
 
 
 class QTests(SimpleTestCase):
+    def test_and_falsey(self):
```

Comment:
Looks like it's passing because of https://github.com/django/django/blob/b3eb6eaf1a197ff155faf333871da032c77ba855/django/utils/tree.py#L95-L104.